forge-gui/res/cardsfolder/upcoming/adrestia.txt

```diff
@@ -0,0 +1,12 @@
+Name:Adrestia
+ManaCost:3
+Types:Legendary Artifact Vehicle
+PT:4/3
+K:Islandwalk
+T:Mode$ Attacks | ValidCard$ Card.Self | IsPresent$ Assassin.CrewedThisTurn | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME attacks, if an Assassin crewed it this turn, draw a card. CARDNAME becomes an Assassin in addition to its other types until end of turn.
```

Comment:
wrong condition

Reply:
I thought the condition is needed for the whole of the trigger (not just the draw) ?

Reply:
what you scripted: "if you control an Assassin that crewed it this turn"

Reply:
Oh right you don't need to control that assassin as this resolves. I guess we need some sort of static "becomescrewed" trigger with remember ?

Reply:
better pass `getCrewedByThisTurn()` into `handlePaid()` inside` AbilityUtils.calculateAmount()`

Reply:
you still need to script intervening if correctly again though